docs/src/enterprise/architecture.md

```diff
@@ -6,6 +6,9 @@ description: lakeFS Enterprise architecture explained!!!!!1
 
 # Architecture
 
+!!! warning
+    ⚠️ fluffy will be deprecated in the upcoming versions and all functionality will be migrated into lakeFS-Enterprise
```

Comment:
- remove the icon the theme already provides one
- lakeFS Enterprise - without the dash